### ISSUE TITLE
[Error: currently only image/png is supported]

### ISSUE THREAD

**Issue by jerryni** [Automattic/node-canvas#481]
I already install canvas with follow command successfully:
`$ npm install canvas --with-jpeg --with-gif`
and [refered to this solution](https://github.com/Automattic/node-canvas/issues/195)

* os: MAC OS 10.10,
* node: v0.10.32

and this is my code:

```
stage.toDataURL({
 mimeType: 'image/jpeg',
 quality: 0.5,
 callback: function(data){}
});

//in the kinetic.js 
toDataURL: function(mimeType, quality) {
            try {
                // If this call fails (due to browser bug, like in Firefox 3.6),
                // then revert to previous no-parameter image/png behavior
                // mimeType = 'image/jpeg';
                // quality = 0.5;
                console.log('mimeType:' + mimeType);
                console.log('lala:' + this._canvas.toDataURL(mimeType, quality));
                return this._canvas.toDataURL(mimeType, quality);
            }
            catch(e) {
                try {
                    //my debug******************Error from here****************************
                    console.log(e);
                    return this._canvas.toDataURL();
                }
                catch(err) {
                    Kinetic.Util.warn('Unable to get data URL. ' + err.message);
                    return '';
                }
            }
        }
```
My ultilmate purpose is speed up the toDataURL callback...

Someone helps me! Thank you first!

**Comment by vishnuvathsan** [comment]
same issue for me too. Im using only PNG images and still getting the error

**Comment by LinusU** [comment]
Closing this since it's an old thread, please see our new installation instructions and report back if you still have problems...

I think that this was fixed by #685
